extensions/wikia/AdEngine/js/lookup/a9.js

```diff
@@ -49,12 +50,8 @@ define('ext.wikia.adEngine.lookup.a9', [
 	function call(skin, onResponse) {
 		var a9Slots;
 
-		trackingOptIn.pushToUserConsentQueue(function(optIn) {
-			if (optIn === false) {
-				log('User opt-out for A9', log.levels.info, logGroup);
-				return;
-			}
-			log('User opt-in for A9', log.levels.info, logGroup);
+		trackingOptIn.pushToUserConsentQueue(function (optIn) {
```

Comment:
This will enable CMP for all users: opted in and out. Do we want to go global or just for opted-in users to first check if everything is fine? Same as for Prebid:
https://github.com/Wikia/app/pull/15449/files#diff-9949d87036c58061a041e2299f5159c3R54